ilc/client/ClientRouter.js

```diff
@@ -274,9 +276,11 @@ export default class ClientRouter extends EventEmitter {
                     if (updateEventsToTrigger.length) {
                         updateEventsToTrigger.forEach(this.emit.bind(this));
                     }
+                    ClientRouter.isRouteChangeStarted = false;
```

Comment:
dont we have some more reliant way, e.g subscribing to internal events? 

also I have small assumption on if we trigger this change multiple times it might lead to inconsistent Router state since we have it as static, @stas-nc @yehor-manzhula @b1ff please validate this assumption

Reply:
I was thinking about subscribing to internal events in CSS tracked app but this method will create additional event listeners on page (as many as we have applications on page). The isRouteChangeStarted static variable shouldn't be inconsistent because we rely on **before router change** and **after route change**  events